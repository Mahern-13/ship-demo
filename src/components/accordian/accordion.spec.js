import React from "react";
import { shallow } from "enzyme";
import Accordion from "./index";
import AccordionItem from "./accordian-item";

describe("Accordion", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Accordion />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should render with one item", () => {
    const wrapper = shallow(
      <Accordion>
        <AccordionItem key={1} />
      </Accordion>
    );
    expect(wrapper).toMatchSnapshot();
  });

  it("should render with multiple items", () => {
    const wrapper = shallow(
      <Accordion>
        <AccordionItem key={1} />
        <AccordionItem key={2} />
      </Accordion>
    );
    expect(wrapper).toMatchSnapshot();
  });

  it("should render when an item is expanded", () => {
    const wrapper = shallow(
      <Accordion>
        <AccordionItem key={1} />
        <AccordionItem key={2} expanded />
      </Accordion>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
