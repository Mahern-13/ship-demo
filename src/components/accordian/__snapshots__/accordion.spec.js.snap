// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion renders the component 1`] = `ShallowWrapper {}`;

exports[`Accordion should render when an item is expanded 1`] = `ShallowWrapper {}`;

exports[`Accordion should render with multiple items 1`] = `ShallowWrapper {}`;

exports[`Accordion should render with one item 1`] = `ShallowWrapper {}`;
